internal/storage/bucket_handle.go

```diff
@@ -64,15 +65,17 @@ func (bh *bucketHandle) BucketType() gcs.BucketType {
 			bh.bucketType = gcs.NonHierarchical
 			return bh.bucketType
 		}
-
+		startTime := time.Now()
+		logger.Infof("\"GetStorageLayout <- (%s)\"", bh.bucketName)
 		storageLayout, err := bh.getStorageLayout()
-
+		duration := time.Since(startTime)
 		// In case bucket does not exist, set type unknown instead of panic.
 		if err != nil {
 			bh.bucketType = gcs.Unknown
 			logger.Errorf("Error returned from GetStorageLayout: %v", err)
 			return bh.bucketType
 		}
+		logger.Infof("\"GetStorageLayout -> (%s)\" %v msec", bh.bucketName, duration.Milliseconds())
```

Comment:
Do we need to explicitly calculate and print duration, when every logger.Infof call anyway prints out time ? Moreover these calls are consecutive, so it should be easy to just manually spot the duration if needed.

Reply:
This log was added here as a part of investigating why mount time for gcsfuse suddenly spiked to 10 sec  from a few msec . While it might be easier to manually spot the duration here, explicitly stating will help in debugging faster when required (Also maintaining consistency with other GCS call logs). Since we discussed [here ](https://github.com/GoogleCloudPlatform/gcsfuse/pull/2629#discussion_r1828007082) that it is not logically correct to log this GCS call at debug_bucket.go , hence the log is added here at info level.